src/drive/locales/en.json

```
@@ -542,55 +543,70 @@
     "doc_type": "Types of document",
     "save": "Save",
     "cancel": "Cancel",
+    "qualify": "Qualify",
+    "apply": "Apply",
     "error": {
       "offline": "You are currently offline and you can't use this functionnality. Try it later",
       "uploading": "You are already uploading a file. Wait until the end of this upload and try again."
     },
     "items": {
-      "national_id_card": "Carte d'identité",
+      "identity": "Identity",
+      "family": "Family",
+      "work_study": "Work",
+      "health": "Health",
+      "home": "Home",
+      "transport": "Transport",
+      "invoice": "Invoice",
+      "others": "Others",
+      "national_id_card": "ID card",
       "passport": "Passeport",
-      "residence_permit": "Titre de séjour",
-      "family_record_book": "Livret de famille",
-      "birth_certificate": "Certificat de naissance",
-      "driver_license": "Permis",
-      "weeding": "Contrat de mariage",
-      "pacs": "Attestion de PACS",
-      "divorce": "Attestation de divorce",
-      "large_family_card": "Carte famille nombreuse",
-      "caf": "CAF",
-      "diploma": "Diplôme",
-      "work_contract": "Contrat",
-      "pay_sheet": "Fiche de paie",
-      "unemployment_benefit": "Allocations chômage",
-      "pension": "Retraite",
-      "other_revenue": "Autres revenues",
-      "gradebook": "Bulletin de notes",
-      "health_book": "Carnet de Santé",
-      "insurance_card": "Carte vitale",
-      "prescription": "Ordonnance",
-      "health_invoice": "Facture médicale",
-      "registration": "Carte grise",
-      "car_insurance": "Assurance voiture",
-      "mechanic_invoice": "Facture de réparation",
-      "transport_invoice": "Facture de transport",
-      "phone_invoice": "Facture de téléphone",
-      "isp_invoice": "Facture de télécom",
-      "energy_invoice": "Fature d'électricité",
-      "web_service_invoice": "Service web",
-      "lease": "Bail",
-      "house_insurance": "Assurance logement",
-      "rent_receipt": "Quitance de loyer"
+      "residence_permit": "Residence permit",
+      "family_record_book": "Family record book",
+      "birth_certificate": "Birth certificate",
+      "driver_license": "Drive license",
+      "weeding": "Weeding contract",
+      "pacs": "Civil union",
+      "divorce": "Divorce",
+      "large_family_card": " Large Family Card",
+      "caf": "Child benefit",
+      "diploma": "Diploma",
+      "work_contract": "Contract",
+      "pay_sheet": "Pay sheet",
+      "unemployment_benefit": "Unemployment benefit",
+      "pension": "Pension",
+      "other_revenue": "Other revenues",
+      "gradebook": "Gradebook",
+      "health_book": "Personal Child Health Record",
```

```suggestion
      "health_book": "Health Record",
```

Je suis pas sur qu'il ya la notion d'enfant dans le carnet de santé?